authors/apps/authentication/tests/test_user_registration.py

```diff
@@ -29,58 +29,92 @@ def setUp(self):
                 "email": "jakey@andela.com",
-                "password": "jakejake"
+                "password": "g@_Gigz-2416"
                 }}
         self.duplicate_email = {
             "user": {
-                "username": "jake",
+                "username": "Gigzjake",
                 "email": "jacky@andela.com",
-                "password": "jakejake"
+                "password": "g@_Gigz-2416"
                 }}
         self.duplicate_email2 = {
             "user": {
-                "username": "jacky2",
+                "username": "jacky2gigz",
                 "email": "jacky@andela.com",
-                "password": "jakejake"
+                "password": "g@_Gigz-2416"
                 }}
-
+        self.username_characters_only = {
+                                            "user": {
+                                                "username": "#@##@#@##@#",
+                                                "email": "jake@jake.jake",
+                                                "password": "g@AHJA473"
+                                                }}
+        self.username_number_only = {
+                                            "user": {
+                                                "username": "123455",
+                                                "email": "jake@jake.jake",
+                                                "password": "g@AHJA473"
+                                                }}
+        self.username_less_than_six = {
+                                            "user": {
+                                                "username": "Mary",
+                                                "email": "jake@jake.jake",
+                                                "password": "g@AHJA473"
+                                            }}
+        self.invalid_password = {
+                                            "user": {
+                                                "username": "MaryWnaja",
+                                                "email": "jake1@jake.jake",
+                                                "password": "gmnbjhggu"
+                                            }}
+        self.invalid_password_less_characters = {
+                                            "user": {
+                                                    "username": "MaryWnaja",
+                                                    "email": "jake1@jake.jake",
+                                                    "password": "@Maty1"
+                                                }} 
+        self.invalid_email = {
+                                            "user": {
+                                                    "username": "MaryWnaja",
+                                                    "email": "jake1jake.jake",
+                                                    "password": "@Maty112Mna"
+                                                }} 
```

Comment:
Leave a blank line before the test method

Reply:
Okay I am working on them